contributors/design-proposals/hpa-v2.md

```diff
@@ -281,7 +281,7 @@ Mechanical Concerns
 The HPA will derive metrics from two sources: resource metrics (i.e. CPU
 request percentage) will come from the
 [master metrics API](resource-metrics-api.md), while other metrics will
-come from the custom metrics API (currently proposed as #34586), which is
+come from the [custom metrics API](custom-metrics-api.md)(currently proposed as #34586), which is
```

Comment:
we can also remove the `(currently proposed as #34586)` bit as that PR was merged to create the file you're linking to.